duckduckgo/iOS
# Hotfix 7.90.1

Task/Issue URL: https://app.asana.com/0/414235014887631/1205575101456370/f
Tech Design URL:
CC: @miasma13 @jaceklyp @Bunn 

**Description**:

Fixes an iOS 15 crash.



**Steps to test this PR**:
1. Update `RemoteMessageRequest.endpoint` with the following entry:

```
return URL(string: "https://raw.githubusercontent.com/duckduckgo/remote-messaging-config/dc43c4359a50e0364002c210beb33f9beb427b8a/live/ios-config/ios-config.json")!
```

Next, run iOS 15, 16, and 17 simulators and verify that all work correctly. You should only see a remote message on 16 and 17, not 15.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
